build-logic-commons/build.gradle.kts

```diff
@@ -0,0 +1,3 @@
+tasks.register("check") {
+    dependsOn(subprojects.map { "${it.name}:check" })
+}
```

Comment:
Still annoying that Gradle doesn't do this automatically.

Can you do the same thing for `test` and `sanityCheck`?

Reply:
This is added here and wired into `sanityCheck`. I think that is all we need for starters.

So now if you or CI runs `sanityCheck` all build logic checks will execute as part of that.

If you want to work with this build on your local machine with individual tasks, you can also `cd <included-build>`. And then run tasks in that build individually.